README.md

```diff
@@ -335,3 +335,62 @@ The response contains the same data as when the repair was created:
 
 As above: if the repair was created without work orders, an empty array of
 workOrders should be returned (rather than no key or a null value)
+
+
+## Get Appointments for Work Order
+
+Returns a list of available appointments for a work order 
+
+```
+GET /v1/workorders/:work_order_reference/appointments/
+```
+
+### Parameters
+
+- Work order reference (required)
+
+### Response
+A successful response should create the Work order in DRS and get the list of available appointments.
+
+```json
+{
+  "metadata": {
+    "resultset": {
+      "count": 5,
+      "offset": 0,
+      "limit": 10
+    },
+  },
+  "results": [
+    {
+	  "beginDate": "2017-10-18T08:00:00Z",
```

Comment:
Looks like the indentation is broken again.

I suggest you set the tab width for markdown to two spaces - here's how: https://msdn.microsoft.com/en-gb/library/ms165330(v=vs.90).aspx#Anchor_1

Reply:
@LBHakairamkonda  I'm sorry, but the indentation is still not correct - what's the best way to sort this out? I can come and sit with you if it helps?